README.md

```diff
@@ -6,7 +6,83 @@
 [![GitHub license](https://img.shields.io/github/license/golemfactory/yapapi)](https://github.com/golemfactory/yapapi/blob/master/LICENSE)
 [![GitHub issues](https://img.shields.io/github/issues/golemfactory/yapapi)](https://github.com/golemfactory/yapapi/issues)
 
-## How to use
+## What's Golem, btw?
+
+[Golem](https://golem.network) is a global, open-source, decentralized supercomputer 
+that anyone can access. It connects individual machines - be that laptops, home PCs or 
+even data centers - to form a vast network, the purpose of which is to provide a way to 
+distribute computations to its provider nodes and allow requestors to utilize its unique 
+potential - which can lie in its combined computing power, the geographical distribution 
+or its censorship resistance.
+
+## Golem's requestor setup
```

Comment:
I'm not 100% sure, but titles are usually capitalized like this: `Golem's Requestor Setup`, not `Golem's requestor setup`.

Reply:
well, that's true for titles ... this is not a title but rather a paragraph heading